### PR TITLE
[NUI] Fix build warning CA1308

### PR DESCRIPTION
**Pull request by bshsqa**
- https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1308
 - CA1308: Normalize strings to uppercase

Signed-off-by: seungho <sbsh.baek@samsung.com>

### CODE REVIEW

File: src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public override object ConvertFromInvariantString(string value)
                     if (head.Length == 2)
                     {
                         useDefault = false;
-                        string radianOrDegree = head[0].Trim().ToLowerInvariant();
+                        string radianOrDegree = head[0].Trim().ToUpperInvariant();
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs:25]: if radianOrDegree is changed as Uppercase then the condition below seems to be changed accordingly.
isn't it like `if (radianOrDegree == "D" || "DEGREE"` 
I am not sure so could you check further?
thank you.

Comment (bshsqa) [reply src/Tizen.NUI/src/internal/XamlBinding/RotationTypeConverter.cs:25]: Thank you. I changed it.

**Review by Seoyeon2Kim — APPROVED** [review]
Looks good to me, but others must approve this PR.
Especially, either @AdunFang  or @elishateng  could check this.

**Comment by dongsug-song** [comment]
@AdunFang 
@elishateng 
could check and review this?
thank you.

**Pull request merged**